library/Icingadb/Web/Navigation/Renderer/HostProblemsBadge.php

```diff
@@ -17,8 +17,12 @@ protected function fetchProblemsCount()
     {
         $summary = HoststateSummary::on($this->getDb())->with('state');
         $this->applyRestrictions($summary);
+        $count = $summary->first()->hosts_down_unhandled;
+        if ($count) {
+            $this->setTitle($count . ' ' . t('unhandled hosts down'));
```

Comment:
Please use `mt('One unhandled host down', '%d unhandled hosts down', $count)`. For the services badge as well please.

Reply:
You mean `tp()` right?

Reply:
Yes, of course.